radare2 installation issues, errors in compilation

Trying to install radare2 errors out with following messages

```

$ sys/install.sh 
WARNING: Updating from remote repository
remote: Enumerating objects: 191, done.
remote: Counting objects: 100% (191/191), done.
remote: Compressing objects: 100% (21/21), done.
remote: Total 236 (delta 175), reused 177 (delta 170), pack-reused 45
Receiving objects: 100% (236/236), 115.66 KiB | 0 bytes/s, done.
Resolving deltas: 100% (183/183), completed with 82 local objects.
From https://github.com/radare/radare2

<<< snip >>>>

CC bin_write_elf.c
CC elf.c
CC elf_write.c
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c: In function ‘populate_relocs_record_from_dynamic’:
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2770:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for (size_t offset = 0; offset < info->rela_size; offset += info->relaent) {
  ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2770:2: note: use option -std=c99 or -std=gnu99 to compile your code
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2776:14: error: redefinition of ‘offset’
  for (size_t offset = 0; offset < info->rel_size; offset += info->relent) {
              ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2770:14: note: previous definition of ‘offset’ was here
  for (size_t offset = 0; offset < info->rela_size; offset += info->relaent) {
              ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2776:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for (size_t offset = 0; offset < info->rel_size; offset += info->relent) {
  ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2782:14: error: redefinition of ‘offset’
  for (size_t offset = 0; offset < info->jmprel_size; offset += size) {
              ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2776:14: note: previous definition of ‘offset’ was here
  for (size_t offset = 0; offset < info->rel_size; offset += info->relent) {
              ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2782:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for (size_t offset = 0; offset < info->jmprel_size; offset += size) {
  ^
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c: In function ‘get_dynamic_info’:
/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.c:2861:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for (size_t i = 0; i < bin->dyn_entries; i++) {
  ^
gmake[3]: *** [/home/flash/R2/radare2/libr/..//libr/bin/p/../format/elf/elf.o] Error 1
gmake[3]: *** Waiting for unfinished jobs....
gmake[2]: *** [bin] Error 2
gmake[1]: *** [all] Error 2
gmake: *** [all] Error 2
$ 
```

Setup and compiler details


```
$ cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.6 (Maipo)
$ 

$ uname  -r
3.10.0-957.el7.x86_64
$ 

$ cc --version
cc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
$ 
```

--- a/libr/bin/format/elf/elf.c
+++ b/libr/bin/format/elf/elf.c
@@ -2767,19 +2767,20 @@ static size_t populate_relocs_record_from_dynamic(ELFOBJ *bin,
 	struct dynamic_relocation_section *info, RBinElfReloc *relocs, size_t pos) {
 	size_t size = get_size_rel_mode(info->plt_mode);
 
-	for (size_t offset = 0; offset < info->rela_size; offset += info->relaent) {
+	size_t offset;
+	for (offset = 0; offset < info->rela_size; offset += info->relaent) {
 		read_reloc (bin, relocs + pos, DT_RELA, info->addr_rela + offset - bin->baddr);
 		fix_rva_and_offset_exec_file (bin, relocs + pos);
 		pos++;
 	}
 
-	for (size_t offset = 0; offset < info->rel_size; offset += info->relent) {
+	for (offset = 0; offset < info->rel_size; offset += info->relent) {
 		read_reloc (bin, relocs + pos, DT_REL, info->addr_rel + offset - bin->baddr);
 		fix_rva_and_offset_exec_file (bin, relocs + pos);
 		pos++;
 	}
 
-	for (size_t offset = 0; offset < info->jmprel_size; offset += size) {
+	for (offset = 0; offset < info->jmprel_size; offset += size) {
 		read_reloc (bin, relocs + pos, info->plt_mode, info->addr_jmprel + offset - bin->baddr);
 		fix_rva_and_offset_exec_file (bin, relocs + pos);
 		pos++;
@@ -2858,7 +2859,8 @@ static RBinElfReloc *populate_relocs_record(ELFOBJ *bin, struct dynamic_relocati
 static struct dynamic_relocation_section *get_dynamic_info(ELFOBJ *bin) {
 	struct dynamic_relocation_section *res = calloc (1, sizeof (struct dynamic_relocation_section));
 
-	for (size_t i = 0; i < bin->dyn_entries; i++) {
+	size_t i;
+	for (i = 0; i < bin->dyn_entries; i++) {
 		Elf_(Dyn) *dyn_struct = bin->dyn_buf + i;
 
 		switch (dyn_struct->d_tag) {

